nextcloud-aio-helm-chart/update-helm.sh

```diff
@@ -407,6 +411,7 @@ sed -i 's|17179869184|"17179869184"|' /tmp/sample.conf
 echo "" >> /tmp/sample.conf
 # shellcheck disable=SC2129
 echo 'STORAGE_CLASS:        # By setting this, you can adjust the storage class for your volumes' >> /tmp/sample.conf
+echo 'STORAGE_CLASS_DATA:        # Allows to set a dedicated storage class for the Nextcloud data volume' >> /tmp/sample.conf
```

Comment:
Maybe we should make the description a bit better by saying the `STORAGE_CLASS` should be a fast storage and `STORAGE_CLASS_DATA` can be a bit slower?